lib/tasks/backfill_in_person_pending_at.rake

```diff
@@ -75,4 +75,62 @@ namespace :profiles do
 
     warn "backfill_in_person_verification_pending_at left #{profiles.count} rows"
   end
+
+  ##
+  # Usage:
+  #
+  # Print pending updates
+  # bundle exec rake profiles:backfill_expired_in_person_verification_pending_at
+  #
+  # Commit updates
+  # bundle exec rake profiles:backfill_expired_in_person_verification_pending_at UPDATE_PROFILES=true
+  #
+  task backfill_expired_in_person_verification_pending_at: :environment do |_task, _args|
+    ActiveRecord::Base.connection.execute('SET statement_timeout = 60000')
+
+    update_profiles = ENV['UPDATE_PROFILES'] == 'true'
+
+    profiles = InPersonEnrollment.where(status: :expired).
+      includes(:profile).
+      where.not(profile: { in_person_verification_pending_at: nil }).
+      map { |enrollment| enrollment.profile }
```

Comment:
The idea with doing it inverted like this is that I'd assume there's much fewer records in the in-person enrollment table than in the profile table, and since we don't have any indices to leverage to optimize the query, this should hopefully be more efficient.

Reply:
if we're worried about query time and table size, we could also try `in_batches` which breaks down the query into batches by ID [(docs link)](https://api.rubyonrails.org/v7.1.3.4/classes/ActiveRecord/Batches.html#method-i-in_batches)

```suggestion
    profiles = InPersonEnrollment.where(status: :expired).
      includes(:profile).
      where.not(profile: { in_person_verification_pending_at: nil }).
      in_batches.flat_map do |batch|
        batch.map { |enrollment| enrollment.profile }
      end
```

Reply:
Checking this with production data, I think it should probably be fine enough. I didn't even need to adjust the default timeout when running test queries.